spec/chrome-fractal-tester.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+const os = require('os');
+const urlParse = require('url').parse;
+const chromeLauncher = require('chrome-launcher');
+const CDP = require('chrome-remote-interface');
+const fractal = require('../fractal');
+
+const HOSTNAME = os.hostname().toLowerCase();
+const REMOTE_CHROME_URL = process.env[ 'REMOTE_CHROME_URL' ];
+
+function launchChromeLocally (headless=true) {
+  return chromeLauncher.launch({
+    chromeFlags: [
+      '--disable-gpu',
+      headless ? '--headless' : '',
+    ],
+  });
+}
+
+function getRemoteChrome () {
+  const info = urlParse(REMOTE_CHROME_URL);
+  if (info.protocol !== 'http:')
+    throw new Error(`Unsupported protocol: ${info.protocol}`);
+  return new Promise(resolve => {
```

Comment:
Hmm, this is just old code that's been moved to a new file, but that's a good point, I'm not sure why this isn't just a `Promise.resolve`! I'll change it.